homeassistant/components/plex/media_player.py

```diff
@@ -90,49 +90,53 @@ def _async_add_entities(
 class PlexMediaPlayer(MediaPlayerEntity):
     """Representation of a Plex device."""
 
+    available = False
+    name = None
+    state = STATE_IDLE
```

Comment:
These three are properties in the base entity class and should remain so here.

Reply:
All of them are properties. But in the end, a property or an instance variable is the same thing for the consuming code. It is faster to access in Python too. I think it's fine.

Reply:
We shouldn't change the interface from the parent class. I want to keep these as properties.

Reply:
Ok.